code/game/objects/items/weapons/melee/misc.dm

```diff
@@ -138,6 +139,7 @@
 	armour_penetration = 1000
 	var/obj/machinery/power/supermatter_shard/shard
 	var/balanced = 1
+	origin_tech = "combat=5;materials=3"

```

Comment:
It does not really matter, but materials 3 is outright pathetic for something that is called "super"matter, lol